source/rooms/RoomTorture.cpp

```diff
@@ -129,7 +129,7 @@ BuildingObject* RoomTorture::notifyActiveSpotCreated(ActiveSpotPlace place, Tile
             Ogre::Real x = static_cast<Ogre::Real>(tile->getX());
             Ogre::Real y = static_cast<Ogre::Real>(tile->getY());
             mCreaturesSpots.emplace(std::make_pair(tile, RoomTortureCreatureInfo()));
-            return loadBuildingObject(getGameMap(), "Roulette", tile, x, y, 0.0, SCALE_ACTIVESPOT_CENTER, false);
+            return loadBuildingObject(getGameMap(), "TortureObject", tile, x, y, 0.0, SCALE_ACTIVESPOT_CENTER, false);
```

Comment:
Well, @bark10 also reported to get his game frozen when he tries to open the settings menu. From what he said, I would guess that the modal window opens but is not visible for some reason. However, I have never seen that and I guess none of us did. If nobody did, we can still release and see if anyone else complains about this.